README.md

```diff
@@ -20,6 +20,32 @@ sequenceDiagram
 ```
 
 
+# Objectives and Explanation of OAuth Status Assertions
+
+Status assertions are used to provide time sensitive claims that impact the validity of digital credentials without the need for a direct request from third parties. They are periodically issued to the holders of the credentials, who can present them to the verifiers together with the corresponding digital credentials.
+
+## Key aspects of the concept of OAuth Status Assertions:
+OAuth Status Assertions provide a robust method of verifying the validity of digital credentials without the need for verifiers to access external sources or third parties.
+
+### Purpose and goal: 
+
+The main goal of OAuth Status Assertions is to **confirm the non-revocation of a digital credential**. This allows verifiers to check the validity of a credential without having to access external systems or databases. This increases privacy, reduces latency and facilitates offline verification.
+
+### Functions and process:
+
+1) **Generation:** A credential issuer generates a status assertion that provides a signed confirmation of the current status of a credential.
+2) **Request and receipt:** The holder of a credential can request a status assertion from the issuer. This is usually done by means of a request containing a JSON Web Token (JWT) or a CBOR Web Token (CWT) that carries specific information about the credential.
+3) **Presentation:** The holder presents the credential together with its status assertion to the verifier. This process is similar to Online Certificate Status Protocol (OCSP) stapling in the TLS protocol, where the server presents a time-stamped certificate status token.
+
+
+### Conceptual features:
+
+**Data protection:** The procedure ensures that no unnecessary personal data of the holder or information about the use of the credential is passed on to the verifier or other parties.
+**Offline capability:** The ability to check the validity of credentials offline without having to access a central database or online status lists is a key advantage of this approach.
+**Independence**: Verifiers do not need direct interaction with the issuer to confirm the validity of a credential, which increases the privacy and security of the process.
```

Comment:
Verifiers do not need direct interaction with the issuer **or other third parties** to confirm the **status** of a credential, which **preservers** the privacy and security of the process.